tf_files/configs/kube-setup-workvm.sh

```diff
@@ -86,25 +89,25 @@ fi
 EOF
   fi
 
-  if ! grep 'kubectl completion bash' ${WORKSPACE}/.bashrc > /dev/null; then 
-    cat - >>${WORKSPACE}/.bashrc << EOF
+  if ! grep 'kubectl completion bash' ${WORKSPACE}/.${RC_FILE} > /dev/null; then 
```

Comment:
This should probably be something like `kubectl completion ${SHELL}`

Reply:
good catch, checking